images/airflow/2.10.1/requirements.txt

```diff
@@ -25,5 +25,7 @@ pyright
 ruff
 sqlalchemy-stubs
 pytest
+pytest-asyncio
```

Comment:
nit: lets keep the scope of this PR limited to 2.9.2, also curious why did we choose to go with 2.9.2 initially?